src/encoded/static/components/static-pages/components/StatisticsPageViewBody.js

```diff
@@ -0,0 +1,1092 @@
+
+            return commonParsingFxn.analytics_to_buckets(resp, 'sessions_by_country', 'ga:country', countKey);
+        }
+    },
+    /*
+    'browse_search_queries' : {
+        'requires' : 'TrackingItem',
+        'function' : function(resp, props){
+            if (!resp || !resp['@graph']) return null;
+
+            var countKey = 'ga:pageviews';
+            if (props.currentGroupBy === 'sessions') countKey = 'ga:users'; // "Sessions" not saved in analytics for search queries.
+
+            return commonParsingFxn.analytics_to_buckets(resp, 'browse_search_queries', 'ga:searchKeyword', countKey);
+        }
+    },
+    'other_search_queries' : {
+        'requires' : 'TrackingItem',
+        'function' : function(resp, props){
+            if (!resp || !resp['@graph']) return null;
+
+            var countKey = 'ga:pageviews';
+            if (props.currentGroupBy === 'sessions') countKey = 'ga:users'; // "Sessions" not saved in analytics for search queries.
+
+            return commonParsingFxn.analytics_to_buckets(resp, 'search_search_queries', 'ga:searchKeyword', countKey);
+        }
+    },
+    */
+    'experiment_set_views' : {
+        'requires' : 'TrackingItem',
+        'function' : function(resp, props){
+            if (!resp || !resp['@graph']) return null;
+
+            //var termBucketField = function(subBucket){ return subBucket['ga:productBrand'] + ' - ' + subBucket['ga:productName']; };
+            const termBucketField = 'ga:productBrand';
+            let countKey = 'ga:productDetailViews';
+
+            if (props.countBy.experiment_set_views === 'list_views') countKey = 'ga:productListViews';
+            else if (props.countBy.experiment_set_views === 'clicks') countKey = 'ga:productListClicks';
+
+            return commonParsingFxn.analytics_to_buckets(resp, 'views_by_experiment_set', termBucketField, countKey);
+        }
+    },
+    /**
+     * For this function, props.currentGroupBy is the interval or time duration, not the actual 'group by' as it is for submissions.
+     * Instead, `props.countBy.file_downloads` is used similar to the google analytics approach.
+     */
+    'file_downloads' : {
+        'requires'  : "TrackingItem", //'TrackingItemDownload',
+        'function'  : function(resp, props){
+            if (!resp || !resp['@graph']) return null;
+            const { countBy : { file_downloads : countBy } } = props;
+
+            let useReport = 'file_downloads_by_experiment_type';
+            let groupingKey = "ga:dimension5"; // Experiment Type
+            const countKey = 'ga:metric2'; // Download Count
+
+            if (countBy === 'filetype'){
+                useReport = 'file_downloads_by_filetype';
+                groupingKey = 'ga:productVariant'; // File Type
+            } else if (countBy === 'geo_country'){
+                useReport = 'file_downloads_by_country';
+                groupingKey = 'ga:country';
```

Comment:
Eventually the entire `aggregationsToChartData` functions I think were supposed to be these hardcoded constants, since the charts and their aggregations and report data structures are theoretically changeable/flexible (i.e. download_tracking items were grabbed using date_histogram, others via regular search). We do have the hardcoded mapping defined in a couple of places like schemas or foursight and maybe could import it here to avoid having to maintain in multiple places...

IMO it'd be better to simply and hardcode more stuff within simplified code, get rid of `aggregationsToChartData` at least in its current form and maybe have more individual React components which each contain their logic.. it would be simpler to understand the code from initial glance for a React dev and get to work on it, whereas with current approach of data coming from various disparate places + objects (aggregationsToChartData, search (now deprecated:) ~~OR histogram~~ response, components themselves, etc.) there is more time/effort needed to understand where data is coming from & what code is doing to make effective changes (arguably more complexity than necessary given context of code --). This code isn't a huge priority (not a view that common users would encounter in normal interaction scenarios / more administrative) so not too much incentive to prioritize yet, will add to backlog.